projects/optic/src/commands/oas/capture.ts

```diff
@@ -193,15 +198,21 @@ export async function captureCommand(): Promise<Command> {
                   feedback.success(`Wrote har to ${outputPath}`);
                   return fs.move(inProgressName, outputPath);
                 }
-                feedback.success(`Wrote har traffic to ${completedName}`);
-                feedback.log(
-                  chalk.gray(
-                    `\nRun ${chalk.whiteBright(
+                if (!enterPressed) {
+                  // Log next steps
+                  feedback.success(`Wrote har traffic to ${completedName}`);
+                  feedback.log(
+                    `\nRun "${chalk.bold(
```

Comment:
- Should we move this feedback below the `verify` running command? So that the next steps are at the bottom
- Should we reword the next step message, since we're already "diffing" traffic here